lib/LDA.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import numpy
+from lib.content_based import ContentBased
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+class LDARecommender(ContentBased):
+    def __init__(self, abstracts, n_factors, n_iterations=5):
+        self.n_factors = n_factors
+        self.n_items = len(abstracts)
+        self.abstracts = abstracts
+        self.n_iterations = n_iterations
+
+    def train(self):
+        tf_vectorizer = CountVectorizer(max_df=0.95, min_df=2, stop_words='english', max_features=self.n_items)
+        tf = tf_vectorizer.fit_transform(self.abstracts)
+
+        lda = LatentDirichletAllocation(n_topics=self.n_factors, max_iter=self.n_iterations,
+                                        learning_method='online', learning_offset=50., random_state=0)
+        self.word_distribution = dist = lda.fit_transform(tf)
```

Comment:
The CountVectorizer still has more options that can be used while computing the term frequencies, like skipping some English words and so on, so probably it's better to use theirs since it can fit more potential future changes.